### PR TITLE
First version, allows a deployment of the repository and basic custom…

### PR DESCRIPTION
**Pull request by lvarin**
…ization.

### CODE REVIEW

File: helm/Chart.yaml
Comment (trispera) [inline helm/Chart.yaml:1]: Could this be replaced with more accurate information regarding this Helm Chart?

**Comment by trispera** [comment]
LGTM. Is there a new version of the `README`?

File: helm/Chart.yaml
Comment (lvarin) [reply helm/Chart.yaml:1]: I will change it

Comment (lvarin) [reply helm/Chart.yaml:1]: changed

**Pull request merged**